migrations/capcons/privatemigration.go

```diff
@@ -25,11 +25,13 @@ import (
 
 func ClearPrivateDomain(
 	inter *interpreter.Interpreter,
+	storageKey interpreter.StorageKey,
 	storageMap *interpreter.StorageMap,
-	storageKey interpreter.StringStorageMapKey,
-	addressPath interpreter.AddressPath,
+	storageMapKey interpreter.StorageMapKey,
 ) {
-	if addressPath.Path.Domain == common.PathDomainPrivate {
-		storageMap.RemoveValue(inter, storageKey)
+	path := storageKeyToPathValue(storageKey, storageMapKey)
+
+	if path.Domain == common.PathDomainPrivate {
```

Comment:
just a nit: Could get just the domain here, instead of creating a new PathValue
```suggestion
	domain := common.PathDomainFromIdentifier(storageKey.Key)
	if domain == common.PathDomainPrivate {
```

Reply:
Improved in 478e9e0ebec1256672483b2b2a35a9b0e10a67d7